pkg/tracingpolicy/generictracingpolicy_test.go

```diff
@@ -527,3 +529,82 @@ func TestYamlNamespaced(t *testing.T) {
 	_, ok := tp.(TracingPolicyNamespaced)
 	require.True(t, ok)
 }
+
+func createTempFile(data string) string {
+	file, err := ioutil.TempFile("", "*")
+	if err != nil {
+		logger.GetLogger().Panic("cannot create temporary file")
+	}
+	path := file.Name()
+
+	_, err = file.WriteString(data)
+	if err != nil {
+		logger.GetLogger().Panicf("cannot write to temporary file %q", path)
+	}
+	err = file.Close()
+	if err != nil {
+		logger.GetLogger().Panicf("cannot close temporary file %q", path)
```

Comment:
maybe you can just `T.Fail` here or just warn since it's not a big deal if we fail to close that file, and should not happen that often.